hackertracker/HTScheduleTableViewController.swift

```diff
@@ -41,10 +41,6 @@ class BaseScheduleTableViewController: UITableViewController, EventDetailDelegat
         if isViewLoaded {
             reloadEvents()
             tableView.scrollToNearestSelectedRow(at: UITableViewScrollPosition.middle, animated: false)
-            // Fix misplaced section header.
```

Comment:
@chrsmys Wait did you need to remove this part?  I am testing and it seems like the only problem was with the dayText